backend/app/views/charities/_charity.json.jbuilder

```diff
@@ -1,3 +1,8 @@
 # frozen_string_literal: true
 
 json.id charity.id
+json.name charity.name
+json.description charity.description
+json.websiteUrl charity.website_url
+json.logoBase64 encoded_file_data_url(charity.logo.attachment)
+json.imageBase64 encoded_file_data_url(charity.image.attachment)
```

Comment:
Do you think it would be good if we just pass e.g. `charity.logo` and `charity.image` to `encoded_file_data_url` and let the function work with the `.attachment`? I think it would be better if we abstract these internals away from the view builders.